### PR TITLE
chore: add volunteer Badbatunde

### PR DESCRIPTION
**Pull request by Badbatunde**

### CODE REVIEW

File: data/users/Badbatunde.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Babatunde",
+  "bio": "Frontend Web dev",
```
Comment (codewithvoid) [inline data/users/Badbatunde.json:3]: nit: 'Frontend Web Developer'

File: data/users/Badbatunde.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://avatars.githubusercontent.com/u/105541302?v=4",
+  "categories": [
+    {
+      "slug": "Web Development",
```
Comment (codewithvoid) [inline data/users/Badbatunde.json:7]: incorrect slug. Correct format
```json
"slug": "web-development"
```

File: data/users/Badbatunde.json (hunk 1 above)
Comment (Badbatunde) [reply data/users/Badbatunde.json]: changes made, kindly check for your further action, thanks.

**Comment by codewithvoid** [comment]
Thanks for making the changes @Badbatunde. Welcome!

You can join community discord [here](https://discord.gg/Tn9E5pn5)

**Pull request merged**